main.go

```diff
@@ -59,14 +64,10 @@ var (
 	defaultKeepAliveDuration = constants.DefaultKeepAliveDuration
 )
 
-func main() {
-	rand.Seed(time.Now().UnixNano())
-
-	klog.InitFlags(nil)
-	ctrllog.SetLogger(klogr.New())
-	if err := flag.Set("v", "2"); err != nil {
```

Comment:
Looks like CAPV was using --v=2 hard-coded before (?). Not sure how exactly that works and if `--v` provided by users was still used. (maybe it just set the default value given that flag parse happens below)

Dropping it should change the amounts of logs emitted by CAPV (per default (?)).

CAPI is running per default with --v=0 at the moment (even though Kuberentes recommends --v=2 for production).

Can't tell what the intention here is, just wanted to surface it.

Reply:
```bash
cluster-api-provider-vsphere on  update-capi-v1.2.0 [$!?] via 🏎💨 v1.18.5
❯ ./capv-manager
F0904 14:55:33.140969   66061 main.go:159] failed to set log level: no such flag -v

cluster-api-provider-vsphere on  update-capi-v1.2.0 [$!?] via 🏎💨 v1.18.5
❯ ./capv-manager -v 2
F0904 14:55:33.140969   66061 main.go:159] failed to set log level: no such flag -v
```
Have been seeing this error when setting the logging verbosity level in the code as well as a command line arg.

Reply:
I think you have to use 2x `-`

```plain
go run main.go --v=10
```
Worked for me